showcase/tests/integration/components/hds/advanced-table/index-test.js

```diff
@@ -175,6 +230,17 @@ const hbsNestedAdvancedTable = hbs`<Hds::AdvancedTable
   </:body>
 </Hds::AdvancedTable>`;
 
+const hbsResizableColumnsAdvancedTable = hbs`<Hds::AdvancedTable
+  @model={{this.model}} @columns={{this.columns}} id="resize-test-table"
```

Comment:
```suggestion
  @model={{this.model}} @columns={{this.columns}} id="resize-test-table" @hasResizableColumns={{true}}
```
[Issue] Now that the resizing has been moved up to the table level it needs to be set here for the tests.